django_project/ford3/templates/user_mgmt/password_reset_confirm_done.html

```diff
@@ -0,0 +1,14 @@
+{% extends "index.html" %}
+
+{% block parent-body %}
```

Comment:
You're using the same template in #436 to reset the password.
This will break.

Reply:
no, it won't. 
althought the template name is the same, but the template for #436 is under `registration` folder, while this one is under `user_mgmt` folder.
perhaps don't use the same template name to avoid confusion?